src/app/HorizontalNav/HorizontalNav.tsx

```diff
@@ -1,72 +1,169 @@
+          itemId={3}
+          isActive={activeItem === 3}
+          to="https://kruizeworkspace.slack.com/"
+        >
           Slack
         </NavItem>
-        <NavItem itemId={3} isActive={activeItem === 3} to="https://www.youtube.com/channel/UCgwhJbGq7NPAkkjgD9cdJXQ">
+        <NavItem
+          itemId={3}
+          isActive={activeItem === 3}
+          to="https://www.youtube.com/channel/UCgwhJbGq7NPAkkjgD9cdJXQ"
+        >
           YouTube
         </NavItem>
-        <NavItem itemId={3} isActive={activeItem === 3} to="https://calendar.google.com/calendar/embed?src=0ccjmebd5q8rgv86ugr8ooc5j0%40group.calendar.google.com&ctz=Asia%2FKolkata">
+        <NavItem
+          itemId={3}
+          isActive={activeItem === 3}
+          to="https://calendar.google.com/calendar/embed?src=0ccjmebd5q8rgv86ugr8ooc5j0%40group.calendar.google.com&ctz=Asia%2FKolkata"
+        >
           Community Call
-          
         </NavItem>
-        <NavItem itemId={3} isActive={activeItem === 3} to="https://calendar.google.com/calendar/embed?src=0ccjmebd5q8rgv86ugr8ooc5j0%40group.calendar.google.com&ctz=Asia%2FKolkata">
+        <NavItem
+          itemId={3}
+          isActive={activeItem === 3}
+          to="https://calendar.google.com/calendar/embed?src=0ccjmebd5q8rgv86ugr8ooc5j0%40group.calendar.google.com&ctz=Asia%2FKolkata"
+        >
           What's New
-          
         </NavItem>
       </NavList>
     </Nav>
-
   );
   const rightnav = (
-    <Avatar
-      className="pf-c-avatar"
-      src={avatarImg}
-      alt="Avatar image"
-    />
+    <React.Fragment>
+      <div style={{ margin: "10px" }}>
+        {avtarImg == "https://www.patternfly.org/v4/images/avatarImg.668560cdf25a4932ef9f711b4acad52d.svg" ?   
+        <Popover
+          aria-label="Basic popover"
+          headerContent={<div>No Minikube Connection.</div>}
+          bodyContent={
+            <div>
+              <b>Minikube connection is pre requisite for autotune setup.</b>
+            </div>
+          }
+        >
+          <Avatar className="pf-c-avatar" src={avtarImg} alt="Avatar" />
+        </Popover>
+        
+        : <Popover
+          aria-label="Basic popover"
+          headerContent={<div>Connected to Minikube Cluster!</div>}
+          bodyContent={
+            <div>
+              <b>Cluster Information </b>
+              <br />
+              Minikube URL : http://{ip}
+              {port == '' ? <></> :
+              <div>
+              <br />
+              Autotune URL : http://{ip}:{port}
+              </div>
+              } 
+              
+            </div>
+          }
+        >
+          <Avatar className="pf-c-avatar" src={avtarImg} alt="Avatar" />
+        </Popover>
+        }
+        
+      </div>
+
+      <Card>
+        <CardBody>
+        
+          {autotuneOn != "" ? (
+            <ProgressStepper isCenterAligned><ProgressStep variant="success" description="Autotune Status"></ProgressStep></ProgressStepper>
```

Comment:
Please check if you can render a simple component like `img` or `icon`. I feel rendering `ProgressStepper` jus for the icon of `success` or `failure` is a costly operation.